aarch64-alpine.dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM balenalib/aarch64-alpine-node:10.15-edge-build
+
+RUN ["cross-build-start"]
+
+RUN apk update && apk add bash gcc g++ ccache git make python linux-headers
```

Comment:
You don't do ``apk update``.
You should probably add ``--no-cache --no-progress`` to ``apk add``.